pkg/daemon/daemon.go

```diff
@@ -266,6 +277,34 @@ func (dn *Daemon) runOnce() error {
 	return fmt.Errorf("%s is not a path nor url; can not run once", dn.onceFrom)
 }
 
+func (dn *Daemon) runFromIgnition() error {
+	if strings.HasPrefix("http://", dn.onceFrom) || strings.HasPrefix("https://", dn.onceFrom) {
+		// TODO(michaelgugino): implement this
+		glog.V(2).Infof("Getting ignition config content from %s", dn.onceFrom)
+		return nil
+	} else if ValidPath(dn.onceFrom) {
+
+		absoluteOnceFrom, err := filepath.Abs(filepath.Clean(dn.onceFrom))
+		if err != nil {
+			return err
+		}
+		ignConfig, err := dn.getIgnitionConfigFromFile(absoluteOnceFrom)
+		if err != nil {
+			return err
+		}
+		// Execute update without hitting the cluster
+		if err := dn.writeFiles(ignConfig.Storage.Files); err != nil {
```

Comment:
Usually after updating files/units, stale data is deleted\config is updated. Do we need to do that type of thing here? cc @ashcrow

Reply:
We should. We can clean this up in a follow on.